src/sentry/api/endpoints/actionable_items.py

```diff
@@ -0,0 +1,127 @@
+from typing import List, Union
+
+from django.utils import timezone
+from drf_spectacular.utils import extend_schema
+from rest_framework.exceptions import NotFound
+from rest_framework.request import Request
+from rest_framework.response import Response
+from typing_extensions import TypedDict
+
+from sentry import eventstore, features
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.helpers.actionable_items_helper import (
+    deprecated_event_errors,
+    errors_to_hide,
+    find_debug_frames,
+    find_prompts_activity,
+    priority_ranking,
+)
+from sentry.api.helpers.source_map_helper import source_map_debug
+from sentry.apidocs.constants import RESPONSE_FORBIDDEN, RESPONSE_NOT_FOUND, RESPONSE_UNAUTHORIZED
+from sentry.apidocs.parameters import EventParams, GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
+from sentry.models import EventError, Organization, Project, SourceMapProcessingIssue
+
+
+class ActionableItemResponse(TypedDict):
+    type: str
+    message: str
+    data: Union[dict, None]
+
+
+class SourceMapProcessingResponse(TypedDict):
+    errors: List[ActionableItemResponse]
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Events"])
+class ActionableItemsEndpoint(ProjectEndpoint):
+    public = {"GET"}
+
+    def has_feature(self, organization: Organization, request: Request):
+        return features.has("organizations:actionable-items", organization, actor=request.user)
+
+    @extend_schema(
+        operation_id="Debug issues related to source maps and event errors for a given event",
```

Comment:
How is this different/related to this [previously documented endpoint](https://docs.sentry.io/api/events/debug-issues-related-to-source-maps-for-a-given-event/)?

The title should be capitalized like `Debug Issues Related ..` (I need to fix that other endpoint I linked :sweat_smile:). Is it also possible to shorten the name to something like `Debug Source Map and Event Errors for an Event`?

Reply:
1. Good catch @schew2381! This would be super confusing with the existing endpoint.
2. Also this endpoint is not actually debugging anything, it just lists the errors to something like: List Event Errors